### PR TITLE
Add prefix to examples in model_doc rst

### PR DESCRIPTION
**Pull request by forest1988**
# What does this PR do?

In my previous PR #11219, I was advised that the example should follow the syntax

```
>>> code_line_1
>>> code_line_2
result
```

I found some `model_doc` rst files that have code blocks without the prefix.
This PR intends to add `>>>` to those examples.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Documentation: @sgugger

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
That's great, thanks you @forest1988

**Comment by sgugger** [comment]
Hi @forest1988, thanks a lot for your PR! In parallel I've merged another PR that enforces proper indentation for those examples so now yours is conflicting. Could you rebase on master and solve the conflicts, or open a new PR from a fresh master, whichever is easier?
Thanks

**Comment by forest1988** [comment]
`check_code_quality` shows the error message as below.
It seems something wrong happens during installing packages.

I'm sorry, but can you try to run circleci again?

```
#!/bin/bash -eo pipefail
pip install .[all,quality]
Defaulting to user installation because normal site-packages is not writeable
Processing /home/circleci/transformers
  Installing build dependencies ... - \ | / done
  Getting requirements to build wheel ... - done
    Preparing wheel metadata ... - \ done
Requirement already satisfied: filelock in /usr/local/lib/python3.6/site-packages (from transformers==4.6.0.dev0) (3.0.12)
Collecting dataclasses
  Using cached dataclasses-0.8-py3-none-any.whl (19 kB)
Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.6/site-packages (from transformers==4.6.0.dev0) (1.7.0)
Collecting tqdm>=4.27
  Using cached tqdm-4.60.0-py2.py3-none-any.whl (75 kB)
Requirement already satisfied: requests in /usr/local/lib/python3.6/site-packages (from transformers==4.6.0.dev0) (2.25.1)
Collecting tokenizers<0.11,>=0.10.1
  Using cached tokenizers-0.10.2-cp36-cp36m-manylinux2010_x86_64.whl (3.3 MB)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from transformers==4.6.0.dev0) (20.9)
Collecting regex!=2019.12.17
  Using cached regex-2021.4.4-cp36-cp36m-manylinux2014_x86_64.whl (722 kB)
Collecting numpy>=1.17
  Using cached numpy-1.19.5-cp36-cp36m-manylinux2010_x86_64.whl (14.8 MB)
Collecting sacremoses
  Using cached sacremoses-0.0.44-py3-none-any.whl
Requirement already satisfied: isort>=5.5.4 in /home/circleci/.local/lib/python3.6/site-packages (from transformers==4.6.0.dev0) (5.8.0)
Collecting black>=20.8b1
  Using cached black-20.8b1-py3-none-any.whl
Collecting flake8>=3.8.3
  Using cached flake8-3.9.0-py2.py3-none-any.whl (73 kB)
Collecting jaxlib>=0.1.59
  Using cached jaxlib-0.1.65-cp36-none-manylinux2010_x86_64.whl (44.7 MB)
Collecting soundfile
  Using cached SoundFile-0.10.3.post1-py2.py3-none-any.whl (21 kB)
Collecting tensorflow>=2.3
  Using cached tensorflow-2.4.1-cp36-cp36m-manylinux2010_x86_64.whl (394.3 MB)
Collecting torchaudio
  Using cached torchaudio-0.8.1-cp36-cp36m-manylinux1_x86_64.whl (1.9 MB)
Collecting Pillow
  Using cached Pillow-8.2.0-cp36-cp36m-manylinux1_x86_64.whl (3.0 MB)
Collecting keras2onnx
  Using cached keras2onnx-1.7.0-py3-none-any.whl (96 kB)
Collecting jax>=0.2.8
  Using cached jax-0.2.12-py3-none-any.whl
Collecting sentencepiece==0.1.91
  Using cached sentencepiece-0.1.91-cp36-cp36m-manylinux1_x86_64.whl (1.1 MB)
Collecting protobuf
  Using cached protobuf-3.15.8-cp36-cp36m-manylinux1_x86_64.whl (1.0 MB)
Collecting flax>=0.3.2
  Using cached flax-0.3.3-py3-none-any.whl (179 kB)
Collecting torch>=1.0

Received "killed" signal
```

File: docs/source/model_doc/bertgeneration.rst [hunk 1]
```diff
@@ -38,38 +38,38 @@ Usage:
...
-    encoder = BertGenerationEncoder.from_pretrained("bert-large-uncased", bos_token_id=101, eos_token_id=102)
-    # add cross attention layers and use BERT's cls token as BOS token and sep token as EOS token
-    decoder = BertGenerationDecoder.from_pretrained("bert-large-uncased", add_cross_attention=True, is_decoder=True, bos_token_id=101, eos_token_id=102)
-    bert2bert = EncoderDecoderModel(encoder=encoder, decoder=decoder)
+    >>> ## leverage checkpoints for Bert2Bert model...
+    >>> ## use BERT's cls token as BOS token and sep token as EOS token
+    >>> encoder = BertGenerationEncoder.from_pretrained("bert-large-uncased", bos_token_id=101, eos_token_id=102)
+    >>> ## add cross attention layers and use BERT's cls token as BOS token and sep token as EOS token
+    >>> decoder = BertGenerationDecoder.from_pretrained("bert-large-uncased", add_cross_attention=True, is_decoder=True, bos_token_id=101, eos_token_id=102)
+    >>> bert2bert = EncoderDecoderModel(encoder=encoder, decoder=decoder)
```
Comment (sgugger) [inline docs/source/model_doc/bertgeneration.rst:46]:
```suggestion
    >>> # leverage checkpoints for Bert2Bert model...
    >>> # use BERT's cls token as BOS token and sep token as EOS token
    >>> encoder = BertGenerationEncoder.from_pretrained("bert-large-uncased", bos_token_id=101, eos_token_id=102)
    >>> # add cross attention layers and use BERT's cls token as BOS token and sep token as EOS token
    >>> decoder = BertGenerationDecoder.from_pretrained("bert-large-uncased", add_cross_attention=True, is_decoder=True, bos_token_id=101, eos_token_id=102)
    >>> bert2bert = EncoderDecoderModel(encoder=encoder, decoder=decoder)
```

File: docs/source/model_doc/bertgeneration.rst [hunk 2]
```diff
@@ -38,38 +38,38 @@ Usage:
...
 
-    # create tokenizer...
-    tokenizer = BertTokenizer.from_pretrained("bert-large-uncased")
+    >>> ## create tokenizer...
```
Comment (sgugger) [inline docs/source/model_doc/bertgeneration.rst:48]:
```suggestion
    >>> # create tokenizer...
```

File: docs/source/model_doc/bertgeneration.rst [hunk 3]
```diff
@@ -38,38 +38,38 @@ Usage:
...
 
-    input_ids = tokenizer('This is a long article to summarize', add_special_tokens=False, return_tensors="pt").input_ids
-    labels = tokenizer('This is a short summary', return_tensors="pt").input_ids
+    >>> input_ids = tokenizer('This is a long article to summarize', add_special_tokens=False, return_tensors="pt").input_ids
+    >>> labels = tokenizer('This is a short summary', return_tensors="pt").input_ids
 
-    # train...
-    loss = bert2bert(input_ids=input_ids, decoder_input_ids=labels, labels=labels).loss
-    loss.backward()
+    >>> ## train...
```
Comment (sgugger) [inline docs/source/model_doc/bertgeneration.rst:54]:
```suggestion
    >>> # train...
```

File: docs/source/model_doc/bertgeneration.rst [hunk 4]
```diff
@@ -38,38 +38,38 @@ Usage:
...
 
 - Pretrained :class:`~transformers.EncoderDecoderModel` are also directly available in the model hub, e.g.,
 
 
 .. code-block::
 
-    # instantiate sentence fusion model
-    sentence_fuser = EncoderDecoderModel.from_pretrained("google/roberta2roberta_L-24_discofuse")
-    tokenizer = AutoTokenizer.from_pretrained("google/roberta2roberta_L-24_discofuse")
+    >>> ## instantiate sentence fusion model
```
Comment (sgugger) [inline docs/source/model_doc/bertgeneration.rst:64]:
```suggestion
    >>> # instantiate sentence fusion model
```

File: docs/source/model_doc/bertweet.rst [hunk 5]
```diff
@@ -31,28 +31,28 @@ Example of use:
...
-    from transformers import AutoModel, AutoTokenizer 
+    >>> import torch
+    >>> from transformers import AutoModel, AutoTokenizer 
 
-    bertweet = AutoModel.from_pretrained("vinai/bertweet-base")
+    >>> bertweet = AutoModel.from_pretrained("vinai/bertweet-base")
 
-    # For transformers v4.x+: 
-    tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base", use_fast=False)
+    >>> ## For transformers v4.x+:
```
Comment (sgugger) [inline docs/source/model_doc/bertweet.rst:39]:
```suggestion
    >>> # For transformers v4.x+: 
```

File: docs/source/model_doc/bertweet.rst [hunk 6]
```diff
@@ -31,28 +31,28 @@ Example of use:
...
+    >>> bertweet = AutoModel.from_pretrained("vinai/bertweet-base")
 
-    # For transformers v4.x+: 
-    tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base", use_fast=False)
+    >>> ## For transformers v4.x+: 
+    >>> tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base", use_fast=False)
 
-    # For transformers v3.x: 
-    # tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base")
+    >>> ## For transformers v3.x:
```
Comment (sgugger) [inline docs/source/model_doc/bertweet.rst:42]:
```suggestion
    >>> # For transformers v3.x: 
```

File: docs/source/model_doc/bertweet.rst [hunk 7]
```diff
@@ -31,28 +31,28 @@ Example of use:
...
+    >>> tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base", use_fast=False)
 
-    # For transformers v3.x: 
-    # tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base")
+    >>> ## For transformers v3.x: 
+    >>> # tokenizer = AutoTokenizer.from_pretrained("vinai/bertweet-base")
 
-    # INPUT TWEET IS ALREADY NORMALIZED!
-    line = "SC has first two presumptive cases of coronavirus , DHEC confirms HTTPURL via @USER :cry:"
+    >>> ## INPUT TWEET IS ALREADY NORMALIZED!
```
Comment (sgugger) [inline docs/source/model_doc/bertweet.rst:45]:
```suggestion
    >>> # INPUT TWEET IS ALREADY NORMALIZED!
```

File: docs/source/model_doc/bertweet.rst [hunk 8]
```diff
@@ -31,28 +31,28 @@ Example of use:
...
 
-    with torch.no_grad():
-        features = bertweet(input_ids)  # Models outputs are now tuples
+    >>> with torch.no_grad():
+    ...     features = bertweet(input_ids)  # Models outputs are now tuples
 
-    ## With TensorFlow 2.0+:
-    # from transformers import TFAutoModel
-    # bertweet = TFAutoModel.from_pretrained("vinai/bertweet-base")
+    >>> ## With TensorFlow 2.0+:
```
Comment (sgugger) [inline docs/source/model_doc/bertweet.rst:53]:
```suggestion
    >>> # With TensorFlow 2.0+:
```

File: docs/source/model_doc/herbert.rst [hunk 9]
```diff
@@ -40,20 +40,20 @@ Examples of use:
...
 
-    encoded_input = tokenizer.encode("Kto ma lepszą sztukę, ma lepszy rząd – to jasne.", return_tensors='pt')
-    outputs = model(encoded_input)
+    >>> encoded_input = tokenizer.encode("Kto ma lepszą sztukę, ma lepszy rząd – to jasne.", return_tensors='pt')
+    >>> outputs = model(encoded_input)
 
-    # HerBERT can also be loaded using AutoTokenizer and AutoModel:
-    import torch
-    from transformers import AutoModel, AutoTokenizer
+    >>> ## HerBERT can also be loaded using AutoTokenizer and AutoModel:
```
Comment (sgugger) [inline docs/source/model_doc/herbert.rst:51]:
```suggestion
    >>> # HerBERT can also be loaded using AutoTokenizer and AutoModel:
```

File: docs/source/model_doc/phobert.rst [hunk 10]
```diff
@@ -31,23 +31,23 @@ Example of use:
...
+    >>> from transformers import AutoModel, AutoTokenizer
 
-    phobert = AutoModel.from_pretrained("vinai/phobert-base")
-    tokenizer = AutoTokenizer.from_pretrained("vinai/phobert-base")
+    >>> phobert = AutoModel.from_pretrained("vinai/phobert-base")
+    >>> tokenizer = AutoTokenizer.from_pretrained("vinai/phobert-base")
 
-    # INPUT TEXT MUST BE ALREADY WORD-SEGMENTED!
-    line = "Tôi là sinh_viên trường đại_học Công_nghệ ."
+    >>> ## INPUT TEXT MUST BE ALREADY WORD-SEGMENTED!
```
Comment (sgugger) [inline docs/source/model_doc/phobert.rst:40]:
```suggestion
    >>> # INPUT TEXT MUST BE ALREADY WORD-SEGMENTED!
```

File: docs/source/model_doc/phobert.rst [hunk 11]
```diff
@@ -31,23 +31,23 @@ Example of use:
...
 
-    with torch.no_grad():
-        features = phobert(input_ids)  # Models outputs are now tuples
+    >>> with torch.no_grad():
+    ...     features = phobert(input_ids)  # Models outputs are now tuples
 
-    ## With TensorFlow 2.0+:
-    # from transformers import TFAutoModel
-    # phobert = TFAutoModel.from_pretrained("vinai/phobert-base")
+    >>> ## With TensorFlow 2.0+:
```
Comment (sgugger) [inline docs/source/model_doc/phobert.rst:48]:
```suggestion
    >>> # With TensorFlow 2.0+:
```

**Review by sgugger — APPROVED** [review]
The failure was just spurious, I relaunched that test and it's passing.
Everything looks good apart from double # in comments, I've made a couple of suggestions to remove those.

**Comment by forest1988** [comment]
Thanks, I applied all suggestions!

I'm sorry, I misunderstood the meaning of the following two that appear in https://huggingface.co/transformers/_sources/quicktour.rst.txt, and assumed that double # were required for comments in the Transformers documentation.
`## PYTORCH CODE` ` ## TENSORFLOW CODE`

If I'm not mistaken in my current understanding, these are special codes to switch between PyTorch and TensorFlow versions, right?

**Comment by sgugger** [comment]
Yes those are special markers, which is why they have the double #, for regular comments, we use just one #

**Pull request merged**